### ISSUE TITLE
Redirect adds `__` within directory path in 3.5.0

### ISSUE THREAD

**Issue by varunpradhan** [cypress-io/cypress#5569]
## Current behavior:

When I login through normal browser then I am able to login.
But when running from cypress test runner the directory path is missing from URL.

**The Cypress Test Runner redirects to : http://<ip>:8084/__/adminHome.html**

![image](https://user-images.githubusercontent.com/26091432/68003324-dde97480-fc92-11e9-880e-7e9e4b4c4cb1.png)

### Desired behavior:

**The Cypress Test Runner should redirect to : http://<ip>:8084/<directory>/adminHome.html**

### Steps to reproduce: (app code and test code)

Below is the code:

```js
cy.visit("http://<ip>:8084/<directory>")
cy.get("#login").click()
cy.get("input[value='Organization']").click()
cy.get("#username").type('<User name provided>')
cy.get("#pwd").type('<password provided>')
cy.get("#login").click()
```

### Versions

Cypress Version : 3.5.0
Operating System : Windows 10
Browser : Chrome 77 / Electron 73

**Comment by varunpradhan** [comment]
It is working only on Electron browser on Cypress version 3.4.1

**Comment by jennifer-shehane** [comment]
Closing as duplicate of https://github.com/cypress-io/cypress/issues/3975

**Comment by jennifer-shehane** [comment]
This fix is available starting in [`4.6.0`](https://docs-staging.cypress.io/guides/references/changelog.html#4-6-0) as an [experiment](https://on.cypress.io/experiments) which you can access by setting this config option in your `cypress.json` or elsewhere:

```json
{
	"experimentalSourceRewriting": true
}
```

The fix is experimental, so there may be some situations where the this is not fixed. 

If you're still this issue while setting the `experimentalSourceRewriting` to true in 4.6.0 - [open a new issue](https://github.com/cypress-io/cypress/issues/new) with a reproducible example + screenshots, etc - filling out our issue template.
